auditlog/models/log.py

```diff
@@ -34,13 +40,31 @@ class AuditlogLogLine(models.Model):
     _description = "Auditlog - Log details (fields updated)"
 
     field_id = fields.Many2one(
-        'ir.model.fields', ondelete='cascade', string=u"Field", required=True)
+        'ir.model.fields', ondelete='set null', string=u"Field", required=True,
+        index=True)
```

Comment:
So there is no problem having a `NOT NULL` constraint (required=True) and `ondelete='set null'`? Is it OK for PostgreSQL?

Reply:
No, thanks for noticing. The required should be dropped (I moved it to check on create and write [here](https://github.com/OCA/server-tools/pull/1262/files#diff-076172f5c3697c2178f79ca37dee1446R60).

Reply:
Ok :) I saw the `create`/`write` overloads too and I did not understand the point while preserving the `required=True`.